run.py

```diff
@@ -164,18 +183,25 @@ def _validate_devices(devices: str):
         print(f"Unable to find model matching {args.model}.")
         exit(-1)
     print(f"Running {args.test} method from {Model.name} on {args.device} in {args.mode} mode.")
-
     # build the model and get the chosen test method
     if args.flops:
         extra_args.append("--flops")
+        extra_args.append(args.flops)
 
     m = Model(device=args.device, test=args.test, jit=(args.mode == "jit"), batch_size=args.bs, extra_args=extra_args)
 
     test = m.invoke
     model_flops = None
+
     if args.flops:
-        assert hasattr(m, "get_flops"), f"The model {args.model} does not support calculating flops."
-        model_flops = m.get_flops()
+        if args.flops == 'model':
+            assert hasattr(m, "get_flops"), f"The model {args.model} does not support calculating flops."
+            model_flops = m.get_flops()
+        else:
+            from components.model_analyzer.TorchBenchAnalyzer import test_dcgm_installation
+            if test_dcgm_installation():
```

Comment:
Here, if user specifies `args.flops == 'dcgm'` and `test_dcgm_installation()` returns False, we should print an error exit immediately.